CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## [2.5.0](https://launchpad.net/mixxx/+milestone/2.4.0) (Unreleased)
```

Comment:
I guess since launchpad is unused for issue tracking, it rather makes sense to link to github milestones now, doesn't it?